microsoft-365/security/defender-endpoint/linux-global-exclusions.md

```diff
@@ -0,0 +1,204 @@
+keywords: microsoft, defender, Microsoft Defender for Endpoint, linux, global exclusions, scans, antivirus
+ms.service: microsoft-365-security
+ms.mktglfcycl: deploy
+ms.sitesec: library
+ms.pagetype: security
+ms.author: ardeshmukh
+author: ardeshmukh
+ms.localizationpriority: medium
+manager: dansimp
+audience: ITPro
+ms.collection: 
+- m365-security
+- tier3
+ms.topic: conceptual
+ms.subservice: mde
+search.appverid: met150
+ms.date: 10/15/2023
+---
+
+# Global Exclusions for Microsoft Defender for Endpoint on Linux
+
+[!INCLUDE [Microsoft 365 Defender rebranding](../../includes/microsoft-defender.md)]
+
+**Applies to:**
+
+- [Microsoft Defender for Endpoint Plan 1](https://go.microsoft.com/fwlink/?linkid=2154037)
+- [Microsoft Defender for Endpoint Plan 2](https://go.microsoft.com/fwlink/p/?linkid=2154037)
+- [Microsoft 365 Defender](https://go.microsoft.com/fwlink/?linkid=2118804)
+
+> Want to experience Defender for Endpoint? [Sign up for a free trial.](https://signup.microsoft.com/create-account/signup?products=7f379fee-c4f9-4278-b0a1-e4c8c2fcdf7e&ru=https://aka.ms/MDEp2OpenTrial?ocid=docs-wdatp-investigateip-abovefoldlink)
+
+This article provides information on how to define Global Exclusions that apply to real-time protection and monitoring.
+
+Global Exclusions are implemented at the level of sensors, by muting the events that match the exclusions. Hence they don't trigger EDR or AV alerts and are excluded from scanning.
+
+> [!NOTE]
+> Apart from real-time protection, Global Exclusions are also applicable to passive mode as sensor events will still be passed to EDR. However they will not be applied during a mdatp custom scan.
+
+![Exclusions checked at every scope ](images/global-exclusions.png)
+
+Global Exclusions can be useful to avoid incorrect detections on files or software that are unique or customized to your organization. They can also be useful for mitigating performance issues caused by Defender for Endpoint on Linux.
+
+> [!WARNING]
+> Defining exclusions lowers the protection offered by Defender for Endpoint on Linux. You should always evaluate the risks that are associated with implementing exclusions, and you should only exclude files that you are confident are not malicious.
+
+## Supported exclusion types
+
+The following table shows the exclusion types supported by Defender for Endpoint on Linux. You can exclude certain files, folders, processes. These exclusions come under the *global* [scope](linux-preferences.md#scopes-of-exclusion-optional).
+
+Exclusion|Definition|Examples
+---|---|---
+File|A specific file identified by the full path|`/var/log/test.log`
+Folder|All files under the specified folder (recursively)|`/var/log/`
+Process|A specific process specified by the full path and all files opened by it|`/bin/cat
+
+> [!IMPORTANT]
+> Global Exclusions does not support wildcard paths.
```

Comment:
Maybe we can call out explicitly that global exclusions are not supported for process name, and extensions as well.

Reply:
I have separate tables, let me know if it still needed.